src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/alerts-classic-vs-workflows.mdx

```diff
@@ -0,0 +1,912 @@
+---
+title: Notification migration 
+metaDescription: Migrating notifications from alerts and applied intelligence 
+redirects: 
+---
+
+At the end of January 2023, the classic notification system for alerts and applied intelligence will be deprecated and we will be implementing the use of workflows instead. This is good news because workflows is an improved, flexible notification system that helps your team learn about potential errors faster so you can take immediate action.
+
+What does [this migration](https://discuss.newrelic.com/t/plan-to-upgrade-alert-notification-channels-to-workflows-and-destinations/188205) mean for your team? In our previous notification model, your team would create an alert condition with different thresholds and parameters. If that condition was associated with a specific policy, was violated, and you wanted your team to know about it right away, you would add **notification settings**. Our notification settings would tell New Relic what data to send where.  
+
+Now, we've added workflows. Workflows are flexible and they don't bind your notification options or your ability to organize alert conditions into polices. For example, you can filter your workflows by a team tag, or check for a "priority" to route issues to the most appropriate destination.
+
+## Configuring custom payloads [#configure-custom-payloads]
```

Comment:
At this point, the reader knows there's a need to move to workflows, but not what they need to do.  It seems like theres a section missing about the migration.

The following data needs to be conveyed somehow:
1. New Relic will automatically migrate accounts from January 9-31 2023
2. New Relic can migrate accounts earlier, just reach out to your account team
3. Consumers can avoid the automated migration by removing channels from policies

It's probably also important to explain what the migration actually does.  It creates a Destination for each Classic Notification Channel and connects them to Workflows created for each Policy.  Only policies with channel associations will be migrated.

The upgrade is accompanied by UI and API changes.  Notification may look slightly different, but should be consistent enough to not break existing integrations.

Reply:
Maybe also add a link to https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/.  It does a good job explaining what workflows and issues.

Reply:
Notification changes:
- All _url attribute values will change and lead to issue based pages, not incident based pages.
- condition_id will now always have the same value as condition_family_id.
- issue_id has been added.  Consumers should switch all integrations to using the issue_id instead of the incident_id, as the incident_id may be removed at some point.
- radar_entity.entityGuid and targets[0].id will be an entity guid when one is available for all types except for Webhooks
- targets[0].labels will contain all tags from the issue, not just the tags for the entity defined by the target.
- targets[0].link and violation_callback_url will lead to the issue page.
- open_violations_count.critical will contain the count of all open incidents across all priorities.  Priority specific counts are unavailable.
- open_violations_count.warning will always be 0.  Priority specific counts are unavailable.
- closed_violations_count.critical will contain the count of all closed incidents across all priorities.  Priority specific counts are unavailable.
- closed_violations_count.warning will always be 0.  Priority specific counts are unavailable.
- owner will have a value of NA if the issue has not been acknowledged